doc/api/cli.md

```diff
@@ -687,3 +694,4 @@ greater than `4` (its current default value). For more information, see the
 [debugger]: debugger.html
 [emit_warning]: process.html#process_process_emitwarning_warning_type_code_ctor
 [libuv threadpool documentation]: http://docs.libuv.org/en/latest/threadpool.html
+[experimental ECMAScript Module]: esm.html#esm_loader_hooks
```

Comment:
Nit: reference list is sorted in ASCII order, so this item needs to be placed before the `[experimental ECMAScript Module]`.

Reply:
Sure, will fix that. Anyway we could add this as a rule to `make lint`?

Reply:
It would be helpful to have it as a rule as we are already a bit messy in some docs in these sections,

Reply:
Would updating [remark-preset-lint-node](https://github.com/watilde/remark-preset-lint-node) to also include [remark-lint-alphabetize-lists](https://www.npmjs.com/package/remark-lint-alphabetize-lists) work? I expect this would cause build failures, if `remark-preset-lint-node` was updated before fixes for that landed in master. How would the transition be handled?

Reply:
I am not sure that `remark-lint-alphabetize-lists` can fix this case as reference lists are not common markdown lists. Maybe @rubys can tell if there is a ready easy solution for this?

Reply:
Remark calls those lines [definitions](https://github.com/nodejs/node/pull/22104#discussion_r210808543).  Looking at the [source to remark-lint-alphabetize-lists
](https://github.com/vhf/remark-lint-alphabetize-lists/blob/master/lib/alphabetical-list-items.js), if `list` where changed to `definition` on [line 18](https://github.com/vhf/remark-lint-alphabetize-lists/blob/master/lib/alphabetical-list-items.js#L18), the code would do what you want.